### ISSUE TITLE
Visual focus indicator missing for mobile 'menu' button (accessibility)

### ISSUE THREAD

**Issue by dandalpiaz** [web-illinois/ilw-header#10]
When the mobile menu button has keyboard focus, there is not a visual focus indicator. The button's background color and text color stay the same when focused and there is not an indicator like an outline/border. Tested on Chrome and Firefox.

Example page: https://builder3.toolkit.illinois.edu/
Example screenshot:

![screenshot of mobile menu button](https://github.com/user-attachments/assets/951d132d-ea39-4f0f-aadb-1041f3e33971)

**Comment by nhjohnstone** [comment]
@bryanjonker-illinois for the focus moving to the close button at the end of the mobile menu, do you want a new issue for that?
